System/Process/Internals.hs

```diff
@@ -24,16 +24,21 @@ module System.Process.Internals (
     PHANDLE, closePHANDLE, mkProcessHandle,
     modifyProcessHandle, withProcessHandle,
     CreateProcess(..),
-    CmdSpec(..), StdStream(..),
+    CmdSpec(..), StdStream(..), ProcRetHandles (..),
     createProcess_,
     runGenProcess_, --deprecated
     fdToHandle,
     startDelegateControlC,
     endDelegateControlC,
     stopDelegateControlC,
+    unwrapHandles,
 #ifndef WINDOWS
     pPrPr_disableITimers, c_execvpe,
     ignoreSignal, defaultSignal,
+#else
```

Comment:
For clarity: could you switch the conditional for `#ifdef WINDOWS`? It's slightly easier to read that way.